rq/command.py

```diff
@@ -72,6 +72,6 @@ def handle_stop_job_command(worker, payload):
     job_id = payload.get('job_id')
     worker.log.debug('Received command to stop job %s', job_id)
     if job_id and worker.get_current_job_id() == job_id:
-        worker.kill_horse()
+        worker.stop_current_job()
```

Comment:
Let's change this to:
```python
worker._stopped_job_id = job_id
worker.kill_horse()
```

Reply:
You're comfortable with accessing a "private" member from outside the class, then?

Reply:
Yes, I'm ok with it. To me, marking something as private means it's not a public API and shouldn't be used by third party code.

This is first party code that's designed for our use and eliminates a network call to Redis, so it's worth it :).